# This patch fixes errors of OpenFOAM-14 in the motorBike tutorial case which
# is used for sanity checks.
#  Andras Horvath (TU-Wien)
diff -Nru OpenFOAM-14-version-14.orig/tutorials/incompressibleFluid/motorBike/motorBike/system/fvSolution OpenFOAM-14-version-14/tutorials/incompressibleFluid/motorBike/motorBike/system/fvSolution
--- OpenFOAM-14-version-14.orig/tutorials/incompressibleFluid/motorBike/motorBike/system/fvSolution	2026-09-23 10:40:39.642269481 +0200
+++ OpenFOAM-14-version-14/tutorials/incompressibleFluid/motorBike/motorBike/system/fvSolution	2026-09-23 10:40:59.272147221 +0200
@@ -57,9 +57,16 @@
 
 relaxationFactors
 {
-    p               0.3;
-    U               0.5;
-    nuTilda         0.5;
+    fields
+    {
+        p               0.3;
+    }
+
+    equations
+    {
+        U               0.5;
+        nuTilda         0.5;
+    }
 }
 
 cache
